MinimedKit/PumpEvents/JournalEntryMealMarkerPumpEvent.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  JournalEntryMealMarkerPumpEvent.swift
+//  RileyLink
+//
+//  Created by Pete Schwamb on 7/14/16.
+//  Copyright © 2016 Pete Schwamb. All rights reserved.
+//
+
+import Foundation
+
+public struct JournalEntryMealMarkerPumpEvent: TimestampedPumpEvent {
```

Comment:
I added this to decocare a while back; I used to use this feature a lot on my pump 😄

There's a carb value here too: https://github.com/bewest/decoding-carelink/blob/313b13b6f465c91d5b01fdaec2b6c4327cabda5f/decocare/history.py#L241